vllm/distributed/device_communicators/tpu_communicator.py

```diff
@@ -18,9 +19,12 @@ def __init__(self, group: ProcessGroup):
             return
         self.disabled = False
 
-        local_rank = dist.get_rank(group)
-        world_size = dist.get_world_size(group)
-        pjrt.initialize_multiprocess(local_rank, world_size)
+        global_rank = dist.get_rank(group)
```

Comment:
add a comment, saying that tpu tensor parallel size are determined at tpu pod slice creation time, and we only support using all of the tpus from a pod slice?

Reply:
@youkaichao Good point. Added the comment. PTAL.